### ISSUE TITLE
Cannot destroy VM if shutdown stucks

### ISSUE THREAD

**Issue by mkuzmin** [jetbrains-infra/packer-builder-vsphere#26]
If shutdown is started, but isn't finished in time (halted for some reason, or timeout is too small) then VM stucks in intermediate state:
It's powered on, but cannot be powered off.
```
==> vsphere: Executing shutdown command...
==> vsphere: Power off VM...
==> vsphere: The attempted operation cannot be performed in the current state (Powered off).
==> vsphere: Destroying VM...
==> vsphere: The attempted operation cannot be performed in the current state (Powered on).
Build 'vsphere' errored: Timeout while waiting for machine to shut down.
```

**Comment by jcoconnor** [comment]
I'm seeing this regularly - also seems to be related to #119

**Comment by mkuzmin** [comment]
seems related to #119 (fixed in v2.0.1)
